### PR TITLE
SAK-30038 Add a webservice to change the active or inactive status of…

### PR DESCRIPTION
**Pull request by ddelblanco**
… a user in a site

It checks the permissions of the session to update or not the site, so doesn't need the superuser check.

### CODE REVIEW

**Comment by ern** [comment]
Hi Diego,
Could you add a unit test for this new webservice?

**Comment by ddelblanco** [comment]
Test scenario:
1. Check the status of a user in the selected site. Usually the user should be active.
2. Call the webservices with rest (it is the easier way to test it) 
2.1 Open a session with: HTTPS://YOURSERVER/sakai-ws/rest/login/login?id=ADMINID&pw=ADMINPASSWORD
2.2 call it: HTTPS://YOURSERVER/sakai-ws/rest/sakai/changeSiteMemberStatus?sessionid=YOURSESSIONID&siteid=THESITEID&eid=THEUSERTOCHANGEEID&active=true (to make the user active or false to make the user inactive)
3. Go to the the site and view if the user is active or not.

**Comment by ddelblanco** [comment]
Ohh... I can view now... unit test!!!  Not asking for how to test. I'll work on it.

**Comment by ddelblanco** [comment]
Added the unit test!

File: webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptChangeSiteMemberStatusTest.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+		// client result
+		String result = client.get(String.class);
+
+		// test verifications
+		assertNotNull(result);
+		assertEquals("success", result);
+	}
+
+	@Test
+	public void testChangeSiteMemberStatusNotExitingUser() {
```
Comment (ern) [inline webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptChangeSiteMemberStatusTest.java]: typo here!

**Comment by ern** [comment]
Nice job thanks Diego

File: webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptChangeSiteMemberStatusTest.java (hunk 1 above)
Comment (ddelblanco) [inline webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptChangeSiteMemberStatusTest.java]: Updated!

**Comment by mateullas** [comment]
Tested on sakai-trunk.atica.um.es.
It works as described.

**Pull request merged**